qpc/messages.py

```diff
@@ -167,11 +167,23 @@
                          'required to merge.'
 REPORT_INVALID_JSON_FILE = 'The file %s does not contain a valid json ' \
                            'details report.'
+REPORT_MISSING_REPORT_VERSION = 'WARNING: '\
+    'The file %s is missing report_version.  '\
+    'Future releases will not tolerate a missing or invalid report_version.'
+REPORT_INVALID_REPORT_TYPE = 'The file %s contains invalid report type %s.  '\
+    'Only details reports can be merged. Excluding from merge.'
 REPORT_JSON_DIR_NO_FILES = \
     'No files with extension .json found in %s.'
 REPORT_VALIDATE_JSON = 'Checking files for valid json details report. %s'
-REPORT_JSON_DIR_FILE_FAILED = 'Failed: %s is not a details report.'
-REPORT_JSON_DIR_FILE_SUCCESS = 'Success: %s is a details report.'
+REPORT_JSON_DIR_FILE_FAILED = 'Failed: %s is not a details report. '\
```

Comment:
Is there any way that we can condense the following 3 error messages?

Reply:
Done